packages/h5wasm/package.json

```diff
@@ -60,6 +60,7 @@
     "eslint": "8.56.0",
     "eslint-config-galex": "4.5.2",
     "react": "18.2.0",
+    "react-dom": "18.2.0",
```

Comment:
I had an error when running the test. `react-dom` is a peer dep of `@h5web/app`, which is a peer dep of `@h5web/h5wasm`, so `react-dom` has to be installed as a dev dep in `@h5web/h5wasm`.